specification/resources/databases/models/pgbouncer.yml

```
@@ -0,0 +1,78 @@
+type: object
+description: PGBouncer connection pooling settings
+
+properties:
+  server_reset_query_always:
+    description: Run server_reset_query (DISCARD ALL) in all pooling modes
+    type: boolean
+    example: false
+  ignore_startup_parameters:
+    description: List of parameters to ignore when given in startup packet
+    type: array
+    example:
+      - extra_float_digits
+      - search_path
+    items:
+      description: Enum of parameters to ignore when given in startup packet
+      type: string
+      enum:
+        - extra_float_digits
+        - search_path
+    maxItems: 32
+  min_pool_size:
+    description: >-
+      Add more server connections to pool if below this number. Improves
+      behavior when usual load comes suddenly back after period of total
+      inactivity. The value is effectively capped at the pool size.
+    type: integer
+    minimum: 0
+    maximum: 10000
+    example: '0'
```

```suggestion
    example: 0
```

`'0'` is a string. removing the commas will make it an integer. There are a few places in this file you'll need to update this.